rfcs/0136-stabilize-incrementally.md

```diff
@@ -0,0 +1,205 @@
+---
+feature: stabilize-incrementally
+start-date: 2022-09-15
+author: John Ericson
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: @thufschmitt @tomberek @infinisil
+shepherd-leader: @tomberek
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Stabilize the new command line interface and Flakes in an incremental fashion, leveraging [RFC 134](https://github.com/NixOS/rfcs/pull/134).
+This plan is designed to still efficiently end the current "limbo" era of these unstable yet widely-used features, yet also soothe the longstanding tensions in the Nix community over how we got here.
+
+# Motivation
+[motivation]: #motivation
+
+For the past few years, we've accumulated a massive backlog of unstable features in the form of the new command-line interface and flakes.
+There is now a growing desire to stabilize those features, but as of yet no plan on exactly how to do so.
+The tension between that desire and a lack of a clear plan has loomed over our heads for a while; this RFC aims to dispel it by providing a concrete plan, an plan that hopefully will mitigate the lingering controversies and tensions around Flakes.
+
+## The situation so far
+
+There are a few facts that must be introduced to give context to this situation.
+We can't change these short of a time machine, so we must find a way to live with them instead.
+
+### Flakes are very popular
+
+As measured in the community polls, Nix has a lot of new users, dwarfing the number of long-time users.
+Flakes is very popular among these new users.
+Ergo, Flakes is very popular among the Nix community as a whole.
+
+Like it or not, these users have been using Flakes as if it was stable, and we cannot make huge drastic changes that would break their code in hard-to-fix ways.
+
+Many Groups and individuals interested in the continued growth the Nix community see Flakes are popular, an also wish it to be stabilized to attract further still more users, since Flakes are already proven to be popular among users.
```

Comment:
```suggestion
Many Groups and individuals interested in the continued growth the Nix community see Flakes are popular, an also wish it to be stabilized to attract still more users, since Flakes are already proven to be popular among users.
```